### PR TITLE
Jenkins: chown after job runs so temp files can be removed, run flake8

### PR DESCRIPTION
**Pull request by elainearbaugh**
@BlaineAtAffirm please review

### CODE REVIEW

File: .jenkins/checks.groovy [hunk 1]
```diff
@@ -82,6 +78,12 @@ freeStyleJob('cabot.docker-compose') { job ->
   jobTemplate(job, "Runs tests against Affirm/cabot.git using docker-compose", 'docker-compose').call()
   publishers {
     archiveJunit('build/docker-compose/*.xml')
+    postBuildScripts {
+      steps {
+        shell('sudo chown -R "$(whoami)" "${WORKSPACE}"')
```
Comment (jimjh) [inline .jenkins/checks.groovy]: :+1:

**Comment by elainearbaugh** [comment]
trigger build

**Comment by BlaineAtAffirm** [comment]
lgtm

btw, in a later PR we may want to do like you almost did in the first PR and use environment variables to change the names for the docker containers. Otherwise, we could get conflicts on parallel builds. Probably won't happen for awhile or at all though

File: .jenkins/checks.groovy [hunk 2]
```diff
@@ -90,7 +92,7 @@ freeStyleJob('cabot.flake8') { job ->
   jobTemplate(job, "Runs tests against Affirm/cabot.git using flake8", 'flake8').call()
   publishers {
     violations {
-      pep8(0, 0, 0, 'build/flake8/flake8.txt')
+      pep8(125, 0, 0, 'build/flake8/flake8.txt')
```
Comment (BlaineAtAffirm) [inline .jenkins/checks.groovy]: btw, I think this is also specified in the ```tox.ini``` we may want to de-dupe that

Comment (elainearbaugh) [inline .jenkins/checks.groovy]: Yeah it is, how would you suggest de-duping it? seems a little silly to have a config file just for this...

**Pull request merged**